wfl/generate/vib.py

```diff
@@ -14,8 +14,8 @@
 from scipy import stats
```

Comment:
let's rename this to `normal_modes.py`?

Reply:
Sure.  I'll do that.

Reply:
Actually, I'm thinking this isn't worth doing here. There will be probably be changes from the deprecation issue, and we need to think about how this should coexist with the (phonopy-based?) version that's applicable to periodic systems.